### PR TITLE
Fix staticcheck failures of pkg/util/ipconfig pkg/util/iptables pkg/util/ipvs/testing

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix staticcheck failures of pkg/util/ipconfig pkg/util/iptables pkg/util/ipvs/testing

**Which issue(s) this PR fixes**:

Ref #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/util/iptables/iptables.go [hunk 1]
```diff
@@ -432,7 +432,7 @@ func iptablesCommand(protocol Protocol) string {
 }
 
 func (runner *runner) run(op operation, args []string) ([]byte, error) {
-	return runner.runContext(nil, op, args)
+	return runner.runContext(context.TODO(), op, args)
```
Comment (SataQiu) [inline pkg/util/iptables/iptables.go]:
```
do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use (SA1012)
```

File: pkg/util/ipconfig/ipconfig.go [hunk 2]
```diff
@@ -42,7 +41,6 @@ const (
 
 // runner implements Interface in terms of exec("ipconfig").
 type runner struct {
-	mu   sync.Mutex
```
Comment (BenTheElder) [inline pkg/util/ipconfig/ipconfig.go]: this is fun, looks like this mutex was never used since the original impl PR introduced it?

Comment (danwinship) [reply pkg/util/ipconfig/ipconfig.go]: Probably the code was originally copied from some other directory. But it looks like no mutex is needed here.

**Pull request merged**